src/pages/cart.js

```diff
@@ -36,99 +36,97 @@ export default function CartPage ({data}){
   return (
     <Layout menu={menu?.data}>
       <section className={cart}>
-  
+        <div className={wrap}>
+          {emptyCart ? (
+            <div className={emptyStateContainer}>
+              <h1 className={title}>Your cart is empty</h1>
+              <p>
+                Looks like you haven’t found anything yet. We understand that
+                sometimes it’s hard to choose — maybe this helps:
+              </p>
+            </div>
+          ) : (
+            <>
+              <h1 className={title}>Your cart</h1>
```

Comment:
just want to confirm on this one that it is fine, what I removed from the previous one didn't exist anyways but want to make sure that nothing else needs to go in there for that to work where it was a joint class name before
Previous:
` <h1 className={[emptyStateHeading, title].join(" ")}>`

Reply:
Yeah this is 100% okay, thank you!